api/openapi-spec/swagger.json

```diff
@@ -4907,6 +4907,9 @@
       "description": "Network name.\nMust be a DNS_LABEL and unique within the vm.\nMore info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names",
       "type": "string"
      },
+     "npwgv1": {
```

Comment:
yes, `npwgv1` stands for that. It was suggested by @dcbw. Lacking a better name, I accepted it myself. I'm fine with using `npwg` until a `v2` is released.

I don't like `standard` because it is even less descriptive and googleable than `npwg`. `defacto` could be nicer